recipes/distromax/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "distromax" %}
+{% set owner = "Rodrigo-Tenorio" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 011c8717d13992cd9a009393bab2e1ee92b5f093cab52ded67037e8cbfdbd7a6
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - numpy
+    - scipy
+    - scikit-image
+    - tqdm
+
+test:
+  imports:
+    - distromax
+
+about:
+  home: https://github.com/{{ owner }}/{{ name }}/
```

Please remove the templating from this URL so that it can be easily copy/pasted rather:
```suggestion
  home: https://github.com/Rodrigo-Tenorio/distromax/
```